### PR TITLE
tests: Small updates to docs

### PR DESCRIPTION
**Pull request by atodorov**
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [x] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: test/README.md [hunk 1]
```diff
@@ -74,6 +74,8 @@ the same architecture, as the one intended for the generated test
 cases. In other words, you need to generate e.g test cases for `aarch64`
 images on an `aarch64` host.
 
+These tests are executed from the `osbuild` repository in GitHub CI.
```
Comment (jrusz) [inline test/README.md:77]: Don't you mean GitLab CI?

**Review by jrusz — CHANGES_REQUESTED** [review]
One minor typo, otherwise looks good.

File: test/README.md (hunk 1 above)
Comment (atodorov) [reply test/README.md:77]: I meant GitHub CI b/c that's the primary trigger for all of the osbuild jobs. Image tests themselves indeed execute in GitLab so not sure what wording to use here.

Comment (jrusz) [reply test/README.md:77]: Well then maybe let's say something like that the tests are triggered by GitHub action which then runs the tests in GitLab CI?

Comment (jrusz) [reply test/README.md:77]: Let's leave it at that and merge this, there is a clear link from Github to Gitlab so this makes sense.

**Review by jrusz — APPROVED** [review]

**Pull request merged**